hardware/arduino/avr/libraries/HID/HID.cpp

```diff
@@ -86,10 +86,13 @@ void HID_::AppendDescriptor(HIDSubDescriptor *node)
 	descriptorSize += node->length;
 }
 
-void HID_::SendReport(uint8_t id, const void* data, int len)
+int HID_::SendReport(uint8_t id, const void* data, int len)
 {
-	USB_Send(pluggedEndpoint, &id, 1);
-	USB_Send(pluggedEndpoint | TRANSFER_RELEASE, data, len);
+    auto ret = USB_Send(pluggedEndpoint, &id, 1);
+    if(ret >= 0){
+        ret += USB_Send(pluggedEndpoint | TRANSFER_RELEASE, data, len);
```

Comment:
Damn how clever. Will you just add a commit and merge it with this one?

The question is still not answered: Why dont we use size_t?